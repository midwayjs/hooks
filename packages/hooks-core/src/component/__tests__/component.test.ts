import { hooks } from '..'

test('component should validate arguments', () => {
  expect(() => hooks({ middleware: null })).toThrowErrorMatchingSnapshot()
})
