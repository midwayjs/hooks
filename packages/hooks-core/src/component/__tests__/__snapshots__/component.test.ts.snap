// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component should validate arguments 1`] = `"[ERR_INVALID_ARG_TYPE]: The 'runtime.middleware' argument must be of type Array. Received null"`;
